dds_web/user.py

```diff
@@ -22,29 +23,48 @@ def login():
     if request.method == "GET":
         if session.get("is_admin"):
             return redirect(url_for("admin.admin_page"))
-        elif session.get("current_user") and session.get("usid"):
+        elif session.get("current_user"):
             return redirect(url_for("user.user_page", loginname=session["current_user"]))
         else:
             return render_template("user/login.html", next=request.args.get("next"))
 
+    # Login form procedure
     if request.method == "POST":
+        # Get username and password from form
         username = request.form.get("username")
         password = request.form.get("password")
-        credentials_validated, is_facility, message, user_info = validate_user_credentials(
-            username, password
-        )
-        if not credentials_validated:
+
+        # Authenticate user
+        try:
+            user_ok = dds_auth.verify_user_pass(username=username, password=password)
+        except sqlalchemy.exc.SQLAlchemyError as sqlerr:
+            # TODO (ina): Create custom error page
+            app.logger.exception(sqlerr)
+            return str(sqlerr), 500
```

Comment:
Could it be meaningful to return in a similar way as in the next except-block?

Reply:
I'm not sure what you mean here, which next except block? Since it's an endpoint we should probably create custom error pages and return those with the exception message, right?